superset/connectors/sqla/models.py

```diff
@@ -551,13 +549,11 @@ class SqlaTable(Model, BaseDatasource):  # pylint: disable=too-many-public-metho
         TableColumn,
         back_populates="table",
         cascade="all, delete-orphan",
-        lazy="selectin",  # Only non-eager loading that works with bidirectional joined.
```

Comment:
The default is `lazy="select"` (which is akin to `lazy=True`), i.e., the relationships are lazy loaded.